packages/deploy/deploy/100_claim/50_multi_claim.ts

```diff
@@ -0,0 +1,30 @@
+import {DeployFunction} from 'hardhat-deploy/types';
```

Comment:
I think we should find a better way to organize our deploy scripts, 
those numbers look random, 50, 60.

Reply:
They must start with a number because hardhat-deploy use that for the execution order. A must is to leave gaps for future scripts. Do you have any idea of a better way ?

Reply:
The condition is to have a number grater than the amount of scripts we expect to add, 50 seems to be enough for me, what number works for you ?

Reply:
In the past we've used multiples of 100, so let's do that